src/lookupStream.js

```diff
@@ -69,6 +70,9 @@ function createPipResolverStream(pipResolver) {
         }
       );
 
+      // prefer a 'postal city' locality when a valid postal code is available
+      usePostalCity( result, doc );
```

Comment:
I like this interface, and it also means that we should be able to wrap this one line in an if statement to enable/disable postal city functionality, right?

Since this code is in a module, if we can configure it with an option like `imports.adminLookup.usePostalCities`, turning this PR into a no-op, then that drastically lowers the barrier to merging it.

Reply:
This has been addressed in https://github.com/pelias/wof-admin-lookup/pull/232/commits/0ef596d87cc052c2fa0219fc34dea658dfd22f9a

The functionality is disabled by default but can be enabled via `imports.adminLookup.usePostalCities`, the only accepted value is `true`.